specs/protocol/derivation.md

```diff
@@ -188,6 +189,15 @@ first block of the epoch appears in the very last L1 block of the window. Note t
 applies to _block_ derivation. Sequencer batches can still be derived and tentatively queued
 without deriving blocks from them.
 
+## Protocol Parameters and Constants
+
+The following table gives an overview of some protocol parameters and constants, and how they are
+affected by protocol upgrades.
+
+| Parameter/Constant | Bedrock (default) value | Latest (default value) | Changes |
+| ------------------ | ----------------------- | ---------------------- | ------- |
+| `max_sequencer_drift` | 600 | 1800 | [Fjord](../fjord/derivation.md#constant-maximum-sequencer-drift) |
```

Comment:
Yeah I meant to differentiate between changeable protocol _parameters_ and unchangeable _constants_, to collect both in that table because, like in the case of the seq drift, we change between parameters and constants. Do we use the word _parameter_ for something unchangeable?

I could change the first column to just read `Parameter` and then have a `Notes` section that explains whether this is a const, or configurable per rollup config, or per L1 SystemConfig, ...